### ISSUE TITLE
(put 'check 'racket-indent-function 1)

### ISSUE THREAD

**Issue by acarrico** [greghendershott/racket-mode#164]
It is a matter of taste, but here is an idea:
  (put 'check 'racket-indent-function 1)
gives:
```
(check my-equal?
   two
   2)
```
instead of:
```
(check my-equal?
       two
       2)
```

**Comment by acarrico** [comment]
I've been using plain *check* a lot more since many of the specializedchecks are broken in typed racket.

**Comment by greghendershott** [comment]
I'm not sure I agree. I have a couple rules of thumb:

1. Usually indent like DrRacket does. (Unless it's obviously wrong, like `for/fold` used to be in older DrRacket).

2. Usually limit special indentation to macros not procedures. (Although yes I realize Scheme/Racket procedures like `with-input-from-file` are exceptions, this rule is usually true.)

3. Also, not a general rule of thumb: In this case it feels weird to tweak `check` but not all the other `check-xxx` procedures.

In short, I think this might be something you should do yourself in your `.emacs`?
